src/App.jsx

```diff
@@ -37,7 +37,12 @@ function App(props) {
     };
   }, []);
 
-  if (!isUserLoading) return <div>LOADING...</div>;
+  if (!isUserLoading)
```

Comment:
Shouldn't this be `if(isUserLoading)`?

Reply:
That was your code but Ill change it
I just get what it means

Reply:
done